### PR TITLE
Support PG 11's new API for savepoint name

### PR DESCRIPTION
**Pull request by jeltz**
PostgreSQL 11 simplified the way to supply the savepoint name to
ReleaseSavepoint() and RollbackToSavepoint() from a list of DefElem to
just a string.

### CODE REVIEW

**Comment by devrimgunduz** [comment]
Is this ready to merge?

**Comment by jeltz** [comment]
In my opinion, but maybe @praiskup wants to take a look first.

File: src/pltrans.c [hunk 1]
```diff
@@ -37,17 +37,18 @@ pl_trans_mark(void *trans)
...
 static DefElem *
-make_defelem(char *name, VALUE arg)
+make_savepont_name(char *name)
 {
     DefElem *f = makeNode(DefElem);
-    f->defname = name;
-    f->arg = (Node *)makeString(RSTRING_PTR(arg));
+    f->defname = "savepoint_name";
+    f->arg = (Node *)makeString(name);
     return f;
```
Comment (praiskup) [inline src/pltrans.c]: This function should return `List *`.

File: src/pltrans.c [hunk 2]
```diff
@@ -37,17 +37,18 @@ pl_trans_mark(void *trans)
     Data_Get_Struct(obj_, struct pl_trans, trans_);                     \
 } while (0)
 
-static char *savename = "savepoint_name";
-
+#if PG_VERSION_NUM >= 110000
+#define make_savepont_name(name) (name)
```
Comment (praiskup) [inline src/pltrans.c]: s/savepont/savepoint/

Comment (jeltz) [reply src/pltrans.c]: Thanks will fix.

File: src/pltrans.c (hunk 1 above)
Comment (jeltz) [reply src/pltrans.c]: Thanks will fix.

**Comment by jeltz** [comment]
I have pushed a new version which fixes both issues. Thanks for the review!

**Comment by praiskup** [comment]
Looks fine to me, thank you.

**Pull request merged**